### PR TITLE
Plasma security now spawn with their gun in the bag

### LINKED ISSUE #4071: Security plasmamen start with no taser
Try to play as security plasmaman. The sec armor gets replaced by the hardsuit and the taser disappears with the plasma tank in its place.

Fixing this should be as easy as moving all the special job stuff to the backpack to avoid conflicts.

### PR DESCRIPTION
**Pull request by pinatacolada**
Fixes #4071

Warden, officers, and pod pilot spawn with the taser in their bag.
Head of Security spawns with egun in the bag.
Brig physician spawns with pen light in their bag.
Also made other species pod pilots spawn with the taser in their bomber
jacket, since it now can hold it, and is more consistent with the rest
of the officers

magistrate looses his armor too, watch out for paper cuts

:cl: pinatacolada
fix: Plasmamen security now spawn with their suit slot items in their
bags
tweak: Pod pilots spawn with taser in the suit slot
tweak: removes armor from plasmamen magistrate suits
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Their helmets should have the WEEWOO thing that can be turned on and off, like justice helmets. :^)

File: code/modules/mob/living/carbon/human/species/plasmaman.dm [hunk 1]
```diff
@@ -58,12 +58,20 @@
...
+		if("Detective")
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/security/
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/security/
-		if("Head of Security", "Magistrate")
+		if("Warden","Security Officer","Security Pod Pilot")
+			suit=/obj/item/clothing/suit/space/eva/plasmaman/security/
+			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/security/
+			H.equip_or_collect(new /obj/item/weapon/gun/energy/advtaser(H), slot_in_backpack)
+		if("Magistrate")
+			suit=/obj/item/clothing/suit/space/eva/plasmaman/security/hos
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: magistrate doesn't need to be armored out the ass.

Comment (TheDZD) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: Or at all really. He is a bureaucrat.

Comment (pinatacolada) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: i didn't change that though

File: code/modules/mob/living/carbon/human/species/plasmaman.dm [hunk 2]
```diff
@@ -58,12 +58,20 @@
 		if("Life Support Specialist")
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/atmostech
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/atmostech
-		if("Warden","Detective","Security Officer","Security Pod Pilot")
+		if("Detective")
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/security/
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/security/
-		if("Head of Security", "Magistrate")
```
Comment (pinatacolada) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: this is the old code

File: code/modules/mob/living/carbon/human/species/plasmaman.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: Oh for heaven's sake. Please give him something else. He absolutely should not have that.

Comment (pinatacolada) [inline code/modules/mob/living/carbon/human/species/plasmaman.dm]: i could probably make a new suit that uses the same sprite but without the armor, I think that's the sensible solution

**Pull request merged**